src/actions/articleAction/articleView.js

```diff
@@ -27,10 +36,27 @@ const fetchArticle = (slug, history) => (dispatch) => {
     });
 };
 
+const fetchFollowers = () => async (dispatch) => {
```

Comment:
Hi @Danielshow. Nice work here. I feel like you can create your own actions file and call your actions from there

Reply:
Calling my actions from here means I am managing `fetchFollowers` state with the result of your actions.  It won't be readable creating another file since it is connected to the same reducer.